.devcontainer/cuda12.0-gcc13/devcontainer.json

```diff
@@ -0,0 +1,66 @@
+{
+  "shutdownAction": "stopContainer",
+  "image": "rapidsai/devcontainers:25.10-cpp-gcc13-cuda12.0",
```

Comment:
We aren't currently building this image in the matrix. I'll add it, this must be fetching an older, stale copy.

Reply:
https://github.com/rapidsai/devcontainers/pull/576